demisto_sdk/commands/common/tools.py

```diff
@@ -1121,26 +1121,42 @@ def find_type_by_path(path: Union[str, Path] = '') -> Optional[FileType]:
 # flake8: noqa: C901
 
 
-def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignore_sub_categories: bool = False):
+def find_type(
+    path: str = '',
+    _dict=None,
+    file_type: Optional[str] = None,
+    ignore_sub_categories: bool = False,
+    ignore_invalid_schema_file: bool = False
+):
     """
-    returns the content file type
+    Returns the content file type.
 
     Arguments:
-        path - a path to the file
+        path (str): a path to the file.
+        _dict (dict): file dict representation if exists.
+        file_type (str): a string representation of the file type.
+        ignore_sub_categories (bool): ignore the sub categories, True to ignore, False otherwise.
+        ignore_invalid_schema_file (bool): whether to ignore raising error on invalid schema files,
+            True to ignore, False otherwise.
 
     Returns:
-        string representing the content file type
+        FileType: string representing of the content file type, None otherwise.
     """
     type_by_path = find_type_by_path(path)
     if type_by_path:
         return type_by_path
+
     try:
         if not _dict and not file_type:
             _dict, file_type = get_dict_from_file(path)
-
     except FileNotFoundError:
         # unable to find the file - hence can't identify it
         return None
+    except ValueError as err:
+        if f'{path} has a structure issue of file type' in str(err) and ignore_invalid_schema_file:
+            # invalid file schema
```

Comment:
Let's add some kind of debug print here

Reply:
ack

Reply:
?

Reply:
acknowledge :)

Reply:
lol yes, I know what ack means, why didn't you add the debug print?

Reply:
Oh I see, I meant to add it after the `if ignore_invalid_schema_file:`

Reply:
done